src/vector/geometry.rs

```diff
@@ -437,11 +437,16 @@ impl PartialEq for Geometry {
 
 impl Eq for Geometry {}
 
+pub fn geometry_type_to_name(ty: OGRwkbGeometryType::Type) -> String {
+    let rv = unsafe { gdal_sys::OGRGeometryTypeToName(ty) };
```

Comment:
Might be worth adding a null check here, even if it's just an assert.

Reply:
Actually, `OGRwkbGeometryType::Type` is an integer, so it's easy to call this with an invalid value. I'd be inclined to return `Option<String>`.

Reply:
Result<String> then? It's more consistent with other APIs in the crate.

Reply:
https://gdal.org/api/vector_c_api.html#_CPPv421OGRGeometryTypeToName18OGRwkbGeometryType says "NULL on failure". Let's do an assertion then.

Got to run now, will do this next week (or feel free to do it yourself if you have time).

Reply:
Done now, and locked down with an extra test.